main.yml

```diff
@@ -935,6 +935,20 @@ storybook:
       - /docs/storybook
   source_repo: https://github.com/RedHatInsights/insights-frontend-storybook
 
+subscription-central:
+  title: Subscription Central
+  channel: '#subscription-central'
+  deployment_repo: https://github.com/RedHatInsights/subscription-central-ui-build
+  frontend:
+    module:
+      appName: subscription-central
+      scope: subscriptionCentral
+      module: ./RootApp
+      group: insights
```

Comment:
No need for this group as it's a helper to idenitfy the app within flat first leven nav.

Reply:
ok removed.